optout/src/main/java/gov/cms/ab2d/optout/OptOutHandler.java

```diff
@@ -3,64 +3,30 @@
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.LambdaLogger;
 import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
-import gov.cms.ab2d.databasemanagement.DatabaseUtil;
-import gov.cms.ab2d.lambdalibs.lib.FileUtil;
 
-import java.io.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
 import java.nio.charset.StandardCharsets;
-import java.nio.file.*;
-import java.sql.Connection;
-import java.util.concurrent.*;
+import java.util.concurrent.CompletionException;
 
 public class OptOutHandler implements RequestStreamHandler {
 
     @Override
     public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context) throws IOException {
         LambdaLogger logger = context.getLogger();
         logger.log("OptOut Lambda is started");
-
-        int threadCount = 2;
-        ExecutorService executorService = Executors.newFixedThreadPool(threadCount);
-        CountDownLatch latch = new CountDownLatch(threadCount);
-
         try {
-            executorService.execute(new OptOutS3(true, latch, logger)); // download file from BFD S3 locally
-            executorService.execute(new OptOutS3(false, latch, logger)); // copy files from BFD S3 to AB2D S3
-
-            latch.await();
-
-            //ToDo: important! add capacity for backpressure
-            BlockingQueue<OptOutMessage> queue = new LinkedBlockingQueue<>();
-
-            Connection dbConnection = DatabaseUtil.getConnection();
-
-            latch = new CountDownLatch(threadCount);
-
-            executorService.execute(new OptOutProducer(OptOutS3.FILE_PATH, queue, latch, logger)); //parse file
-            executorService.execute(new OptOutConsumer(queue, dbConnection, latch, logger)); //update database
-
-            latch.await();
-        } catch (NullPointerException | InterruptedException | CompletionException ex) {
+            //ToDo: Get file name from SQS
+            OptOutProcessing optOutProcessing = new OptOutProcessing("optOutDummy.txt", logger);
+            optOutProcessing.process();
+        } catch (NullPointerException | CompletionException ex) {
```

Comment:
Dont we need to account for "InterruptedException" anymore? if thread is interrupted, either before or during the activity.

Reply:
InterruptedException could occur due to multithreading (Producer, Consumer). Currently we only have one main thread and this type of exception will never be thrown.